sdk/nodejs/sqs/sqsMixins.ts

```diff
@@ -104,7 +104,7 @@ function createFunctionFromEventHandler(
     if (handler instanceof Function) {
         return new lambda.CallbackFunction(name, {
             callback: handler,
-            policies: [iam.ManagedPolicy.AWSLambdaFullAccess, iam.ManagedPolicy.AmazonSQSFullAccess]
+            policies: [iam.ManagedPolicy.AmazonSQSFullAccess]
```

Comment:
Similar to my comment about Kinesis. So I think we should clean this up by encapsulating the full copy of the policy somewhere common to refer to, then refer to it here, in the Kinesis mixin, and in the Lambda mixin.